### DISCUSSION TITLE
Why there is not a ReadOnlySequence<T>.PositionOf(ReadOnlySpan<T> span) extension method?

### DISCUSSION THREAD

**Discussion by GF-Huang** [dotnet/runtime#48275] [Q&A]
Sometimes we have to find `\r\n` rather than only  `\n`.

**Comment by davidfowl** [comment]
Because the SequenceReader has methods to find all kinds of methods to find text. Once you get the ReadOnlySequence, construct a SequenceReader from it and it should make your life easier
